samples/transient-clients/Core_6/Client/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```

Comment:
Yeah, that is why I asked for his review. I am sure I made lots of these types of mistakes :)